modules/python/Makefile.security

```diff
@@ -0,0 +1,15 @@
+BANDIT=$(call which, bandit)
+BANDIT_FLAGS=-ll -ii
+
+PYTHON_LINT_TARGETS?=$(shell find . -name "*.py" -not -path "*/.venv/*" -not -path "*/$(BUILD_HARNESS_VENV_NAME)/*" -not -path "*/.tox/*" -not -path "*/.egg*" -not -path "*/frontend/*" -not -path "*/dist/*" -not -path "*/build/*" -not -path "*/tests/*")
+
+
+.PHONY: python/security/targets
+python/security/targets:
+	@VAR=( $(PYTHON_LINT_TARGETS) ); echo "Discovered $${#VAR[@]} files:"
+	@echo "$$(echo $(PYTHON_LINT_TARGETS) | tr ' ' '\n')"
+
+.PHONY: python/bandit
+python/bandit: bh/venv reports/ python/security/targets
```

Comment:
Sorry to drag this out, but I was just thinking about how we have python/lint and python/fmt, and the libraries themselves are python/lint/isort, for example. In that light, could we name this python/scan/bandit, and add a python/scan target that runs all of our canonical code scanning, similar to how linting and formatting are arranged?

Maybe something like this?
```makefile
python/security/targets:
python/security/announce/%
python/security/bandit
python/security: python/security/bandit
python/scan: python/security
```

Reply:
I'm going to add a security scanning stage to our jenkins build that runs after post-build tests which calls `make scan`

Reply:
That sounds reasonable to me.

Regarding the jenkins build stage, is it wise to enable this that widely for everyone? Can you make the output of a stage not fail the build since I would be surprised if there weren't problems in quite a few repositories?

Reply:
This would act like the testing stages, in that it would run the make command only if it exists in your makefile. So you'll have to add `scan:` to any repo you want to enable that stage for.

Reply:
ref https://gitlab.com/mintel/everest/jsl/-/merge_requests/101